### PR TITLE
ref: remove elasticsearch initialization

### LINKED ISSUE #6712: Remove elasticsearch initialization
elasticsearch is neither working nor supported. It causes confusion and incompatibilities for beginners. Hence, it's integration should be removed. For now, remove any iniitalization but keep the integration code.

However, in future as well, we should not use a service which requires 64 GB of RAM to run (sarcasm)

### PR DESCRIPTION
**Pull request by codedsun**
Fixes #6712 

#### Short description of what this resolves:

ref: remove elasticsearch initialization

#### Checklist

- [x] I have read the [Contribution & Best practices Guide](https://blog.fossasia.org/open-source-developer-guide-and-best-practices-at-fossasia) and my PR follows them.
- [x] My branch is up-to-date with the Upstream `development` branch.
- [x] The unit tests pass locally with my changes 
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have added necessary documentation (if appropriate)

- [x] All the functions created/modified in this PR contain relevant docstrings.

### CODE REVIEW

**Comment by codedsun** [comment]
@iamareebjamal  - check.

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity decreasing per file
==============================
+ app/instance.py  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4756484&bid=15862981)

**Comment by iamareebjamal** [comment]
There's more code adding scheduled jobs for elasticsearch

**Comment by codedsun** [comment]
@iamareebjamal - When the base condition of `app.config['ENABLE_ELASTICSEARCH']` is false, it wont work because I removed from env file and in the config - if the env variable is not false then by default it's false. 

Also, the issue mentions `For now, remove any iniitalization but keep the integration code.`

**Comment by iamareebjamal** [comment]
> because I removed from env file and in the config

OK, how will it get removed from my .env file when I pull latest changes?

**Comment by codedsun** [comment]
No it wont be removed from you env, then why not only read from config directly as you issue says to keep the integrated code.? So whenever we want it back, we dont have to do much work.

@iamareebjamal

**Comment by iamareebjamal** [comment]
I never asked to remove the integration code. Read my comment again, carefully

https://github.com/fossasia/open-event-server/pull/6713#issuecomment-570320014

**Comment by codedsun** [comment]
@iamareebjamal - issue states
`For now, remove any iniitalization but keep the integration code.`..


> I never asked to remove the integration code. Read my comment again, carefully
> 
> [#6713 (comment)](https://github.com/fossasia/open-event-server/pull/6713#issuecomment-570320014)

No problem sir. I will remove it.

**Comment by iamareebjamal** [comment]
> issue states
For now, remove any initialization but keep the integration code

Yes, and that's what is to be done.

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity decreasing per file
==============================
+ app/instance.py  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4756484&bid=15862981)

File: app/instance.py [hunk 1]
```diff
@@ -251,10 +240,10 @@ def update_sent_state(sender=None, headers=None, **kwargs):
...
 # scheduler.add_job(send_mail_to_expired_orders, 'interval', hours=5)
 # scheduler.add_job(empty_trash, 'cron', hour=5, minute=30)
-if app.config['ENABLE_ELASTICSEARCH']:
-    from app.views.elastic_cron_helpers import sync_events_elasticsearch, cron_rebuild_events_elasticsearch
-    scheduler.add_job(sync_events_elasticsearch, 'interval', minutes=60)
-    scheduler.add_job(cron_rebuild_events_elasticsearch, 'cron', day=7)
+# if app.config['ENABLE_ELASTICSEARCH']:
+#     from app.views.elastic_cron_helpers import sync_events_elasticsearch, cron_rebuild_events_elasticsearch
+#     scheduler.add_job(sync_events_elasticsearch, 'interval', minutes=60)
+#     scheduler.add_job(cron_rebuild_events_elasticsearch, 'cron', day=7)
```
Comment (iamareebjamal) [inline app/instance.py]: Delete the code, don't comment

This is a git project, not a college project shared in pen drives

**Comment by niranjan94** [comment]
Codacy Here is an overview of what got changed by this pull request:


```diff

Issues
======
+ Solved 1
           

Complexity decreasing per file
==============================
+ app/instance.py  -3
         
```


See the complete overview on [Codacy](https://app.codacy.com/manual/fossasia/open-event-orga-server/pullRequest?prid=4756484&bid=15862981)

**Pull request merged**